internal/framework/service/api_token_permissions_groups/data_source_test.go

```diff
@@ -0,0 +1,104 @@
+package api_token_permissions_groups
```

Comment:
this is why you are getting cyclic import issues. the test packages are suffixed with `_test`

```suggestion
package api_token_permissions_groups_test
```

Reply:
Thanks!